main.py

```diff
@@ -63,22 +110,38 @@ def performEvaluation(issue):
     if commit_hash:
         checkout_commit(commit_hash, input_dir)
 
-    success = run_specimin(specimin_command)
+    specimin_command = build_specimin_command(issue_id, issue[JsonKeys.ROOT_DIR.value], issue[JsonKeys.PACKAGE.value], issue[JsonKeys.TARGETS.value])
+
+    success = run_specimin(specimin_command, os.path.join(issue_folder_dir, specimin_project_name))
```

Comment:
How are you testing this? Put another way, have you checked that this code can successfully check out Daikon and run Specimin on it? If so, it would be a good idea to include a test script here and hook it up to CI right away, so that we have evidence that the code is working.

Reply:
This PR is successfully checking out Daikon and ruuning Specimin on it.
However, this PR does not include the CF command that generates the respective CF issue (crash/FP). To do this, dockerization or other method should be adopted since that requires a specific version of Java and CF

Reply:
That shouldn't prevent us from writing a smoke test that just verifies that specimin ran and exited successfully. Can you please add one (checking just the error code is fine). As a matter of good software engineering practice, it is always better to have a test rather than no tests :)